docs/user_manual/components.md

```diff
@@ -127,6 +127,12 @@ If `i_n` is not provided, `loading` of line will be a `nan` value.
 | `tan0` | `double`  | -          | zero-sequence shunt loss factor (tan𝛿)     | &#10024; only for asymmetric calculations | &#10060; |                                   |
 | `i_n`  | `double`  | ampere (A) | rated current                              |                 &#10060;                  | &#10060; |               `> 0`               |
 
+```{note}
```

Comment:
Lets probably mention this in the calculation page of short circuit. Does not fit under the input of node title.

Reply:
Or maybe we can modify the required column instead

Reply:
what do you propose?

Reply:
I do believe that this should be mentioned here as well. Otherwise, not all info will be present where expected. I also added an entry for the short circuit calculations section with this node.

Reply:
You are right. It would kind of seem odd in the short circuit calculation page. My concern was that there too many notes below the tables around the page which makes it less readable, plus its mentioned twice.

 Technically, the required column mentions `&#10024; only for asymmetric calculations` which should be self sufficient. Just `&#10024;` in table and `&#10024; : Only for asymmetric power-flow and short-circuit calculations` below the table is also a possibility. What do you  think?

Reply:
Or we leave it as you originally did and deal with notes later if they become a big deal, apologies for this nitpick :D